client-tags/typing.md

```diff
@@ -27,14 +27,14 @@ For the purpose of this specification, a message is defined as a user-generated
 ## Format
 The `typing` tag is sent as a [`TAGMSG`][tags]. The client tag takes no parameters.
```

Comment:
For consistency with other client tag specs, how about this:

```
A typing notification is represented by a [`TAGMSG`][tags] command sent with a tag using the client-only prefix `+` and no value:

    +draft/typing
```

We can then refer to just "typing notifications" without backticks in the rest of the spec.